config/environments/production.rb

```diff
@@ -49,7 +49,7 @@
   config.log_level = ENV.fetch('RAILS_LOG_LEVEL', 'info')
 
   # Prevent health checks from clogging up the logs.
-  # config.silence_healthcheck_path = "/up"
+  config.silence_healthcheck_path = '/status'
```

Comment:
I forgot to uncomment the line! Just added to #728. Glad I saw this in your PR before the other got merged. 🙏🏻